Demo/app/src/main/java/com/juanocampo/swipeable/swipeablelist/swpeable/adapter/SwipeableHelperAdapter.java

```diff
@@ -75,18 +77,23 @@ public final void onBindViewHolder(final RecyclerView.ViewHolder holder, int pos
      * @see RecyclerView.ViewHolder#getAdapterPosition()
      */
     public void onItemDismiss(int position) {
-        listener.swiped(position);
+        int visiblePositionSelected = position - manager.findFirstVisibleItemPosition();
```

Comment:
Because we need to store just the visibile position to avoid trigger the wrong position. as i explain you above.